### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jh7494**

### CODE REVIEW

**Comment by tomasort** [comment]
I added some more detail to the readme

File: README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
-### Running tests:
-
-To run the tests, use `nosetests`
+### Instructions on running tests:
```
Comment (jh7494) [inline README.md]: The phrase "Instructions on" is redundant. See https://github.com/nyu-devops/lab-flask-tdd
@yaozcoderepo I recommend the original section headline as it is more concise.

Comment (yaozcoderepo) [reply README.md]: No problem. Feel free to change it back @jh7494

**Review by tinay0313 — APPROVED** [review]

**Review by tomasort — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -71,16 +71,26 @@ The HTTP response, in this case would be:
     "ongoing": true,
     "product_id": 1,
     "start_date": "01-01-2022 10:10:10 ",
-    "type": "Value",
+    "type": "VALUE",
```
Comment (jh7494) [inline README.md:74]: @tomasort I believe "Value" is a hardcoded string which indicates "Value" in a Enum (line 49 in service/models.py) and the type in json object will be deserialized at line 155. I don't think "Value" is equivalent to "VALUE".

Comment (jh7494) [reply README.md:74]: I just realized the latest commit has changed all constants in upper-case. Never mind.

Comment (tomasort) [reply README.md:74]: Yes, you are right. I thought I changed it. For some reason, it worked the other day, but I tried the same thing today, and as you said, 'Value' was not recognized.

**Review by yaozcoderepo — APPROVED** [review]

**Pull request merged**